### ISSUE TITLE
[Feature] Discounts on products

### ISSUE THREAD

**Issue by loulou133** [Ctrlpanel-gg/panel#1044]
### Feature Description

Add to your version 1.0 the possibility of getting discounts on a product (a server) when you order

### Solution Idea

When you go to the “Servers” category and click on create a server, you choose your server and at the end, there will be a summary of what it will cost and if you want to put a coupon code if there is one.

**Comment by S0ly** [comment]
Coupon already exist ?

**Comment by loulou133** [comment]
> Coupon already exist ?

No, it doesn't exist. There is a discount on purchasing products in the store, but not on creating servers. if people already have credits without having bought them, or earn credits in a different way than buying, so that everyone can have a promotion for the products or a specific product

**Comment by S0ly** [comment]
its actually true, no discount can be applied to servers, also I think it does not make lots of sense to apply a coupon to credit purchases
